modules/common/pod/pod.go

```diff
@@ -47,3 +47,21 @@ func GetPodListWithLabel(
 
 	return podList, nil
 }
+
+// GetHeadlessPodSvcName gets a list of pods matching the labels provided and returns a slice of pod and service names. This
+// is required in order to allow resolution of pods behind Headless services.
```

Comment:
So does the value returned here (e.g. `horizon-memcached-0.horizon-memcached`) can be used to connect to the pod? So is this the hostname of the Pod or the FQDN of the Pod? I'm asking this as I would like to find a name for this function that better describes what it returns.

Reply:
It's the FQDN of the pod. So, you're suggesting something like: `GetHeadlessPodFQDN()` ?

Reply:
`GetHeadlessPodFQDN` sounds good to me